auth_helper.py

```diff
@@ -0,0 +1,68 @@
+import os
+import pathlib
+
+import json5
+import sys
+import requests
+import msal
+
+class AuthHelper:
+    tenantId = os.environ["tenant_id"]
+    scope = os.environ["scope"]
+    authority = os.environ["authority"]
+    appId = os.environ["appId"]
+    accessToken = ""
+    def __init__(self,**kwargs):
+        for key,value in kwargs.items():
+            setattr(self,key,value)
+
+    def adoAuthenticate(self):
```

Comment:
Use `snake_casing` and don't forget about return types.

Reply:
Resolved.